PhysicsTools/NanoAOD/python/nanoDQM_cff.py

```diff
@@ -18,6 +18,131 @@
+    Plot1D('iEtaMod5', 'iEtaMod5', 20, 0, 40, 'iEtaMod5'),
+    Plot1D('iEtaMod20', 'iEtaMod20', 20, 0, 40, 'iEtaMod20'),
+    Plot1D('iPhiMod2', 'iPhiMod2', 20, 0, 199, 'iPhiMod2'),
+    Plot1D('iPhiMod20', 'iPhiMod20', 100, 0, 99, 'iPhiMod20')
+])
+
+_Photon_extra_plots = cms.VPSet()
+for plot in nanoDQM.vplots.Photon.plots:
+    _Photon_extra_plots.append(plot)
+_Photon_extra_plots.extend([
+    Plot1D('r9Frac', 'r9Frac', 22, 0, 1.1, 'Fractional R9'),
+    Plot1D('superclusterEta', 'superclusterEta', 30, 3.0, 3.0, 'supercluster Eta'),
+    Plot1D('energy', 'energy', 20, 0, 80, 'energy'),
+    Plot1D('rawPreshowerEnergy', 'rawPreshowerEnergy', 20, 0, 80, 'rawPreshowerEnergy'),
+    Plot1D('seedClusEnergy', 'seedClusEnergy', 20, 0, 40, 'seedClusEnergy'),
+    Plot1D('e5x5', 'e5x5', 20, 0, 20, 'E5x5'),
+    Plot1D('dEtaSeedClusSuperClus', 'dEtaSeedClusSuperClus', 20, 0, 1.0, 'dEtaSeedClusSuperClus'),
+    Plot1D('dPhiSeedClusSuperClus', 'dPhiSeedClusSuperClus', 20, 0, 1.0, 'dPhiSeedClusSuperClus'),
+    Plot1D('sigmaIphiIphiFull5x5', 'sigmaIphiIphiFull5x5', 20, 0, 0.1, 'sigmaIphiIphi Full5x5'),
+    Plot1D('eMax', 'eMax', 20, 0, 40, 'eMax'),
+    Plot1D('e2nd', 'e2nd', 20, 0, 40, 'e2nd'),
+    Plot1D('eTop', 'eTop', 20, 0, 40, 'eTop'),
+    Plot1D('eBottom', 'eBottom', 20, 0, 40, 'eBottom'),
+    Plot1D('eLeft', 'eLeft', 20, 0, 40, 'eLeft'),
+    Plot1D('eRight', 'eRight', 20, 0, 40, 'eRight'),
+    Plot1D('e2x5Top', 'e2x5Top', 20, 0, 40, 'e2x5Top'),
+    Plot1D('e2x5Bottom', 'e2x5Bottom', 20, 0, 40, 'e2x5Bottom'),
+    Plot1D('e2x5Left', 'e2x5Left', 20, 0, 40, 'e2x5Left'),
+    Plot1D('e2x5Right', 'e2x5Right', 20, 0, 40, 'e2x5Right'),
+    Plot1D('nSaturatedXtals', 'nSaturatedXtals', 10, 0, 9, 'number of Saturated crystals'),
+    Plot1D('numberOfClusters', 'numberOfClusters', 10, 0, 9, 'number of Clusters'),
+    Plot1D('hadTowOverEm', 'hadTowOverEm', 20, 0, 0.2, 'Single Tower H/E'),
+    Plot1D('ecalRecHitIsolation', 'ecalRecHitIsolation', 20, 0, 40, 'ecal RecHit Isolation'),
+    Plot1D('sigmaIetaIetaFrac', 'sigmaIetaIetaFrac', 20, 0, 0.08, 'sigmaIetaIetaFrac'),
+    Plot1D('ecalPFClusterIso', 'ecalPFClusterIso', 20, 0, 40, 'ecal PF Cluster Isolation'),
+    Plot1D('hcalPFClusIso', 'hcalPFClusIso', 20, 0, 40, 'hcal PF Cluster Isolation'),
+    Plot1D('chargedHadronIso', 'chargedHadronIso', 20, 0, 40, 'chargedHadronIso'),
+    Plot1D('trkSumPtSolidConeDR04', 'trkSumPtSolidConeDR04', 20, 0, 40, 'trkSumPtSolidConeDR04'),
+    Plot1D('iEtaMod5', 'iEtaMod5', 20, 0, 40, 'iEtaMod5'),
+    Plot1D('iEtaMod20', 'iEtaMod20', 20, 0, 40, 'iEtaMod20'),
+    Plot1D('iPhiMod2', 'iPhiMod2', 20, 0, 199, 'iPhiMod2'),
+    Plot1D('iPhiMod20', 'iPhiMod20', 100, 0, 99, 'iPhiMod20')
+])
+
+from Configuration.ProcessModifiers.egmCustomNano_cff import egmCustomNano
+(egmCustomNano).toModify(
+    nanoDQM.vplots.Electron,
+    plots = _Electron_extra_plots
+)
+(egmCustomNano).toModify(
+    nanoDQM.vplots.Photon,
+    plots = _Photon_extra_plots
+)
+
```

Comment:
These can be removed (and so can `egmCustomNano_cff.py`) and I believe the additional plots can be initialized from within the `addExtraEGammaVarsCustomize` function in `egamma_custom_cff.py` by doing something like:

https://github.com/RSalvatico/cmssw/blob/45caabcfd5167ed973ffb15df03759de176779f3/PhysicsTools/NanoAOD/python/egamma_custom_cff.py#L7-L18

I admit though that I am not sure how to toggle the EGM customization for nanoDQM the same way as NANO:@EGM. Something as trivial as @nanoAODDQM:@EGM does not work, and I don't know if it is even supposed to. Perhaps @vlimant can shed some light on this.